projects/ngx-datatable/src/lib/components/body/body.component.ts

```diff
@@ -666,58 +670,21 @@ export class DataTableBodyComponent<TRow extends Row = any> implements OnInit, O
   };
 
   /**
-   * Calculates the styles for the row so that the rows can be moved in 2D space
-   * during virtual scroll inside the DOM.   In the below case the Y position is
-   * manipulated.   As an example, if the height of row 0 is 30 px and row 1 is
-   * 100 px then following styles are generated:
-   *
-   * transform: translate3d(0px, 0px, 0px);    ->  row0
-   * transform: translate3d(0px, 30px, 0px);   ->  row1
-   * transform: translate3d(0px, 130px, 0px);  ->  row2
-   *
-   * Row heights have to be calculated based on the row heights cache as we wont
-   * be able to determine which row is of what height before hand.  In the above
-   * case the positionY of the translate3d for row2 would be the sum of all the
-   * heights of the rows before it (i.e. row0 and row1).
-   *
-   * @returns the CSS3 style to be applied
+   * Calculates the offset of the rendered rows.
+   * As virtual rows are not shown, we have to move all rendered rows
+   * by the total size of previous non-rendered rows.
+   * If each row has a size of 10px and the first 10 rows are not rendered due to scroll,
+   * then we have a renderOffset of 100px.
    */
-  rowsStyles = computed(() => {
-    const rowsStyles: NgStyle['ngStyle'][] = [];
-    this.rowsToRender().forEach((rows, index) => {
-      const styles: NgStyle['ngStyle'] = {};
-
-      // only add styles for the group if there is a group
-      if (this.groupedRows) {
-        styles.width = this.columnGroupWidths.total;
-      }
-
-      if (this.scrollbarV && this.virtualization) {
-        let idx = 0;
-
-        if (Array.isArray(rows)) {
-          // Get the latest row rowindex in a group
-          const row = rows[rows.length - 1];
-          // The group row, which has always a numeric index
-          idx = row ? this.getRowIndex(row).index : 0;
-        } else {
-          if (rows) {
-            // normal rows always have a numeric index
-            idx = this.getRowIndex(rows).index;
-          } else {
-            // When ghost cells are enabled use index to get the position of them
-            idx = this.indexes().first + index;
-          }
-        }
-
-        // const pos = idx * rowHeight;
-        // The position of this row would be the sum of all row heights
-        // until the previous row position.
-        styles.transform = `translateY(${this.rowHeightsCache().query(idx - 1)}px)`;
-      }
-      rowsStyles.push(styles);
-    });
-    return rowsStyles;
+  renderOffset = computed(() => {
+    if (this.scrollbarV && this.virtualization) {
+      // const pos = idx * rowHeight;
```

Comment:
```suggestion
```

Reply:
Removed the entire comment, as everything relevant is in the docs comment above.